pyprof/nvtx/nvmarker.py

```diff
@@ -528,6 +528,14 @@ def patch_model_configs():
     patch_never_call(torch.autograd, "gradgradcheck", "gradgradcheck")
     patch_never_call(torch.autograd.profiler.record_function, "__init__", "record_function")
 
+    # Patch both AMP libraries
+    #
+    import importlib
+    if importlib.util.find_spec("apex.amp") is not None:
+        import apex.amp
```

Comment:
If you are patching both versions of the library here, do we still need to patch amp_C?

Reply:
Can we move this to the patch apex method and remove the amp_C version?

Reply:
This code isn't patching the apex classes.  This code is only capturing one function (the one that enables AMP) from each library and using it to determine if AMP was enabled.

I don't know enough about what the existing amp_C patching does to remove it. Based on my understanding we shouldn't need to patch it (or apex.amp) since we still end up capturing all of the leaf torch functions that pass through amp, but I'm not inclined to remove it without fully understanding what it does and why it was added.